gems/quilt_rails/README.md

```diff
@@ -38,6 +38,8 @@ A turn-key solution for integrating Quilt client-side libraries into your Rails
 
 ## Server-side-rendering
 
+🗒 The quick start guide is for Shopify internal only assuming the use of [`sewing-kit`](https://github.com/Shopify/sewing-kit) and [`dev`](https://github.com/Shopify/dev)
```

Comment:
We can't enforce "Shopify internal only" on a public repo. It's probably better for this to be worded something like

* The quick start guide is authored for internal Shopify developers with access to `@shopify/sewing-kit` and `dev`

Reply:
I would say this is a big struggle while doing this update. A lot of the setup depends on sewing-kit and dev, working on a version of the doc & tool that is good of the for public only was very very complex and likely cause more confusion then needed.

I like your wording thou 👍

Reply:
IMO, it's a little distracting to call out internal vs external in different parts of the doc.  If possible, get it out of the way early, like:

> This guide is focused on internal Shopify developers with access to `dev` and `@shopify/sewing-kit`.  A similar setup can be achieved using `quilt_rails`, and following the [`react-server-webpack-plugin`](../../packages/react-server-webpack-plugin/README.md) guide. Apps not running on Shopify infrastructure _must_ disable server-side GraphQL queries.